### PR TITLE
Reduce scanner/queue load

### PR DESCRIPTION
**Pull request by tbg**
This change is [Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5251)

### CODE REVIEW

**Comment by bdarnell** [comment]
store.go change LGTM, but as discussed offline (and in #5252)  let's remove the gc_queue.go change and instead modify the thresholds on the previous two lines

**Comment by tschottdorf** [comment]
Done (and tests fixed), PTAL

File: storage/gc_queue_test.go [hunk 1]
```diff
@@ -91,22 +91,22 @@ func TestGCQueueShouldQueue(t *testing.T) {
...
 		// x2 GC'able bytes, avg age = TTLSeconds.
-		{2 * bc, 2 * bc * ttl, 0, 0, now, true, 2},
+		{2 * bc, 2 * bc * ttl, 0, 0, now, true, 2 * considerThreshold},
 		// GC'able bytes, intent bytes, and intent normalization * 2 elapsed.
 		// Queues solely because of gc'able bytes.
-		{bc, 5 * bc * ttl, 10 * ia, 0, now, true, 5},
+		{bc, 5 * bc * ttl, 10 * ia, 0, now, true, 5 * considerThreshold},
 		// A contribution of 1 from gc, 10/5 from intents.
-		{bc, bc * ttl, 5, 10 * ia, now, true, 1 + 2},
+		{bc, bc * ttl, 5, 10 * ia, now, true, (1 + 2) * considerThreshold},
```
Comment (bdarnell) [inline storage/gc_queue_test.go]: You could move these multiplications down to the point where you compare against `test.priority` so it's not repeated on every line.

**Comment by bdarnell** [comment]
LGTM

File: storage/gc_queue_test.go (hunk 1 above)
Comment (tschottdorf) [inline storage/gc_queue_test.go]: Done.

**Pull request merged**